core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -360,6 +361,11 @@ public void setCompressor(Compressor compressor) {}
 
       @Override
       public void setDecompressor(Decompressor decompressor) {}
+
+      // TODO(lukasz) should we return something here?
```

Comment:
Alternatively, if we put REMOTE_ADDR_KEY in GprcUtil, service could check the type of the value to tell whether it's a network-borne or local request.